components/organisation-unit-tree/src/organisation-unit-node/use-org-children.test.js

```diff
@@ -0,0 +1,330 @@
+import { CustomDataProvider } from '@dhis2/app-runtime'
+import { renderHook } from '@testing-library/react-hooks'
+import React from 'react'
+import { useOrgChildren } from './use-org-children.js'
+
+describe('OrganisationUnitTree - useOrgChildren', () => {
+    // @TODO: Figure out why this is necessary at all...
+    const origError = console.error
+    jest.spyOn(console, 'error').mockImplementation((...args) => {
```

Comment:
What's the error you're getting here? Because it might be that you're getting errors logged from react-query's logger (though hthat should be disabled).

Reply:
I don't know if it's any help, and I don't see any obvious differences, but these tests aren't generating any warnings: https://github.com/dhis2/app-runtime/blob/master/services/data/src/react/hooks/useDataQuery.test.tsx. Maybe there's something useful there.

Reply:
I was getting a log with "Warning: An update to TestComponent inside a test was not wrapped in act(...)."
I added something to the first `it` and removed the mocking